### ISSUE TITLE
Plone 5: "Actions" menu in toolbar making any sense?

### ISSUE THREAD

**Issue by zopyx** [plone/Products.CMFPlone#725]
Choose: 'copy' on an content object from within the folder_contents view.
Then you can "paste" it directly using the copy button. However "paste" action remains unavailable
from the Actions toolbar menu (usability fail).

**Comment by thet** [comment]
I agree, that providing copy/paste/delete from the folder_contents view might be enough.
But I use the actions menu quite a lot for custom actions. Also lineage 2.0 uses it for marking a content object as a childsite.

**Comment by zopyx** [comment]
The core problem is that the actions is not being updated. That is the usability fail

Regards
Andreas Jung
-----
Sorry for being brief - sent from a mobile device.

**Comment by vangheem** [comment]
Yes, this is now a side affect of showing toolbar actions on folder contents. We're going to have to reload the toolbar after every copy and cut action.

**Comment by davilima6** [comment]
Or just update `Paste` buttons states via AJAX after `Copy` and `Cut` on both Folder Contents and the Toolbar.
